test/testdata/generate.sh

```diff
@@ -132,16 +135,17 @@ scaffold_test_project project-v2 --project-version=2
 # [Currently, default CLI plugin] - Project version 3 (default) uses plugin go/v3 (default).
 scaffold_test_project project-v3
 scaffold_test_project project-v3-multigroup
-scaffold_test_project project-v3-declarative-v1 --plugins="go/v3,declarative,grafana/v1-alpha"
+scaffold_test_project project-v3-declarative-v1 --plugins="go/v3,declarative,grafana/v1-alpha,monitoring/v1-alpha"
 scaffold_test_project project-v3-config --component-config
 scaffold_test_project project-v3-with-deploy-image
 scaffold_test_project project-v3-with-metrics
+scaffold_test_project project-v3-with-monitoring
```

Comment:
Could you please add this new plugin to the metrics sample with seems the category releated and allow us to check only the diff scaffolds that will be done by it?